firmware/controllers/system/efi_gpio.cpp

```diff
@@ -79,8 +79,10 @@ EnginePins::EnginePins() {
 #if EFI_PROD_CODE
 #define setPinValue(outputPin, electricalValue, logicValue)                        \
   {                                                                                \
-    if ((outputPin)->currentLogicValue != (logicValue)) {                          \
-	  palWritePad((outputPin)->port, (outputPin)->pin, (electricalValue));         \
+    if ((outputPin)->currentLogicValue != (logicValue)) {	\
+	  brain_pin_e tmppin = (outputPin)->brainPin;		\
+	  const char* tmpmsg = (outputPin)->m_msg;		\
```

Comment:
I have _zero_ clue what's going on here. Like _no_ clue whatsoever. That's a red flag.

Reply:
To be fair, I barely changed anything.